src/Electron/electron.ts

```diff
@@ -119,7 +119,7 @@ export class ElectronMessageBus implements MessageBus {
 /**
  * @extends ContainerBase
  */
-export class ElectronContainer extends ContainerBase {
+export class ElectronContainer extends WebContainerBase {
```

Comment:
WebContainerBase simply adds the saving of a globalWindow property which can be provided via DI or is set at run-time by looking for the global window object.  This will be defined in the renderer process but remain null in the main process.